src/data/repositories/AlertSyncDataStoreRepository.ts

```diff
@@ -0,0 +1,125 @@
+import { D2Api } from "@eyeseetea/d2-api/2.36";
+import { DataStoreClient } from "../DataStoreClient";
+import { Future } from "../../domain/entities/generic/Future";
+import {
+    AlertSyncOptions,
+    AlertSyncRepository,
+} from "../../domain/repositories/AlertSyncRepository";
+import { FutureData } from "../api-futures";
+import { getOutbreakKey, getValueFromMap } from "./utils/AlertOutbreakMapper";
+import { Maybe } from "../../utils/ts-utils";
+import { DataSource } from "../../domain/entities/disease-outbreak-event/DiseaseOutbreakEvent";
+import { DataValue } from "@eyeseetea/d2-api/api/trackerEvents";
+import { OptionsD2Repository } from "./OptionsD2Repository";
+import { Attribute, D2TrackerTrackedEntity } from "@eyeseetea/d2-api/api/trackerTrackedEntities";
+import {
+    Alert,
+    AlertSynchronizationData,
+    OutbreakType,
+} from "../../domain/entities/alert/AlertData";
+
+export type AlertData = {
+    alert: D2TrackerTrackedEntity;
+    attribute: Maybe<Attribute>;
+    dataSource: DataSource;
+};
+
+export class AlertSyncDataStoreRepository implements AlertSyncRepository {
+    private dataStoreClient: DataStoreClient;
+    private optionsRepository: OptionsD2Repository;
+
+    constructor(private api: D2Api) {
+        this.dataStoreClient = new DataStoreClient(api);
+        this.optionsRepository = new OptionsD2Repository(api);
```

Comment:
Code smell calling one repository from another repository, you should call diferent repositories in the use case